protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -655,6 +655,130 @@ definitions:
             }
         type: object
         existingJavaType: com.fasterxml.jackson.databind.JsonNode
+      oauth_connector_input_specification:
```

Comment:
@bazarnov any utility in adding the state generation configuration we talked about now?

Or do we not have an example of strict state format requirements? making generation configurable likely useless?

Reply:
Done.